src/main/java/ua/goit/timonov/enterprise/module_6_2/view/console/ConsoleInput.java

```diff
@@ -0,0 +1,140 @@
+package ua.goit.timonov.enterprise.module_6_2.view.console;
+
+import java.util.*;
+
+import static ua.goit.timonov.enterprise.module_6_2.view.console.ConsolePrinter.printLine;
+
+/**
+ * Provides input of different data types from console: String, int, float, Date
+ */
+public class ConsoleInput {
+
+    public static final int MIN_YEAR_VALUE = 1900;
+    public static final int MIN_MONTH_VALUE = 1;
+    public static final int MAX_MONTH_VALUE = 12;
+    public static final int MIN_DAY_VALUE = 1;
+    public static final int DAYS_IN_LONG_MONTH = 31;
+    public static final int DAYS_IN_SHORT_MONTH = 30;
+    public static final int DAYS_IN_LONG_FEBRUARY = 29;
+    public static final int DAYS_IN_SHORT_FEBRUARY = 28;
+    public static final int FOUR = 4;
+    public static final int HUNDRED = 100;
+    public static final int FOUR_HUNDRED = 400;
+
+    /* list of month's numbers with 31 days in that month */
+    private static List<Integer> longMonths = Arrays.asList(1, 3, 5, 7, 8, 10, 12);
+
+    /* list of month's numbers with 30 days in that month */
+    private static List<Integer> shortMonths = Arrays.asList(4, 6, 9, 11);
+
+    /**
+     * inputs String from console
+     * @return      inputted string
+     */
+    public static String inputString() {
+        Scanner sc = new Scanner(System.in);
+        return sc.nextLine();
+    }
+
+    /**
+     * inputs int value from console
+     * @return      inputted int value
+     */
+    public static int inputInteger() {
+        while(true) {
+            try {
+                Scanner sc = new Scanner(System.in);
+                return sc.nextInt();
+            }
+            catch (Exception e) {
+                printLine("Wrong input! Please, input an integer value.");
+            }
+        }
+    }
+
+    /**
+     * inputs int value from console, inputted value must be in defined interval including bounds
+     * @param lowBound          interval's low bound
+     * @param highBound         interval's high bound
+     * @return                  inputted int value
+     */
+    public static int inputInteger(int lowBound, int highBound) {
+        while(true) {
```

Comment:
move all while-trues to one method called "mainloop". use this methods for "single" read